modules/kubernetes-addons/kafka/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for strimzi-kafka-operator.
+
+# Default replicas for the cluster operator
+replicas: 1
+
+# If you set `watchNamespaces` to the same value as ``.Release.Namespace` (e.g. `helm ... --namespace $NAMESPACE`),
+# the chart will fail because duplicate RoleBindings will be attempted to be created in the same namespace
+watchNamespaces: []
+watchAnyNamespace: true
+
+defaultImageRegistry: quay.io
+defaultImageRepository: strimzi
+defaultImageTag: 0.31.1
```

Comment:
We should remove this since they are the chart defaults. It makes version changes easier for users 

https://github.com/strimzi/strimzi-kafka-operator/blob/main/helm-charts/helm3/strimzi-kafka-operator/values.yaml

Reply:
I removed the defaults except the resources section which are modified